docs/guides/getting-started/concepts.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Concepts
+description: 'Get familiar with the terminology used in Latitude'
+---
+
+## Prompts
+
+Prompts are the core building block of Latitude. They are the questions or tasks that you want to ask your model. Prompts can be as simple as a single sentence or as complex as a multi-step dialogue. You can use prompts to generate text, classify text, or perform any other task that your model is capable of.
+
+In Latitude, prompts support parameters, chaining, shared snippets, and more advanced features like logic (if/else, loops) or version control.
+
+To find out more about prompts, check out the [Prompt manager](/guides/prompt-manager/) overview.
```

Comment:
Aren't we calling them `Documents`?

Reply:
yeah i also think publicly it makes more sense to call them prompts